core/transport/src/grpc/catalog/catalog_master.proto

```diff
@@ -202,6 +202,7 @@ message Layout {
   optional string layoutType = 1;
   optional LayoutSpec layoutSpec = 2;
   optional bytes layoutData = 3;
+  map<string, ColumnStatisticsInfo> stats = 4;
```

Comment:
is `ColumnStatisticsInfo` a generalized structure, or specific to hive?

Reply:
it came from hive, but i think it is quite generic.